### DISCUSSION TITLE
The widget background does not change.

### DISCUSSION THREAD

**Discussion by Wazzap84** [NightscoutFoundation/xDrip#3591] [Bug reports] [answered]
On the old version, the widget background was made transparent. In the new version, nothing happens when the background is changed.

**Comment by Navid200** [comment]
Thanks for reporting.

Would you please identify the last Nightly release that worked as you expect?

**Reply by Wazzap84** [reply]
xDrip-plus-20231114

**Reply by Navid200** [reply]
So, you can confirm that you have tested xDrip-plus-20231121 and it fails?

**Reply by Wazzap84** [reply]
Fail in xDrip-plus-20240419-645e8b4 vesion.

**Reply by Wazzap84** [reply]
After the last server failure, Google switched to the latest version.

**Reply by Navid200** [reply]
So, xDrip-plus-20231114 works.
But, xDrip-plus-20240419-645e8b4 fails.  Is that it?  

There are 25 Nightly releases between those two.
Can you find which one is that one that works?  

That will make it easier for us to see what change in code may have caused it.

**Reply by Wazzap84** [reply]
Okay, today I'll start rolling back and write in which version it stopped working.

**Reply by Wazzap84** [reply]
I'm very sorry. I was changing the wrong parameter. Everything works. Once again, I'm sorry to bother you.

**Reply by Navid200** [reply, marked as answer]
No worries.
I'm glad it is working

If you ever need assistance with xDrip, please kindly open a discussion instead of an issue.  If needed, we can always convert it to an issue.
